packages/astro/src/types/public/config.ts

```diff
@@ -1328,58 +1328,57 @@ export interface ViteUserConfig extends OriginalViteUserConfig {
 
 		/**
 		 * @docs
-		 * @name image.experimentalLayout
+		 * @name image.layout
 		 * @type {ImageLayout}
 		 * @default `undefined`
 		 * @description
 		 * The default layout type for responsive images. Can be overridden by the `layout` prop on the image component.
-		 * Requires the `experimental.responsiveImages` flag to be enabled.
 		 * - `constrained` - The image will scale to fit the container, maintaining its aspect ratio, but will not exceed the specified dimensions.
 		 * - `fixed` - The image will maintain its original dimensions.
 		 * - `full-width` - The image will scale to fit the container, maintaining its aspect ratio.
 		 */
-		experimentalLayout?: ImageLayout | undefined;
+		layout?: ImageLayout | undefined;
 		/**
 		 * @docs
-		 * @name image.experimentalObjectFit
+		 * @name image.objectFit
 		 * @type {ImageFit}
 		 * @default `"cover"`
 		 * @description
 		 * The default object-fit value for responsive images. Can be overridden by the `fit` prop on the image component.
-		 * Requires the `experimental.responsiveImages` flag to be enabled.
+		 * Requires a value for `layout` to be set.
 		 */
-		experimentalObjectFit?: ImageFit;
+		objectFit?: ImageFit;
 		/**
 		 * @docs
-		 * @name image.experimentalObjectPosition
+		 * @name image.objectPosition
 		 * @type {string}
 		 * @default `"center"`
 		 * @description
 		 * The default object-position value for responsive images. Can be overridden by the `position` prop on the image component.
-		 * Requires the `experimental.responsiveImages` flag to be enabled.
+		 * Requires a value for `layout` to be set.
 		 */
-		experimentalObjectPosition?: string;
+		objectPosition?: string;
 		/**
 		 * @docs
-		 * @name image.experimentalBreakpoints
+		 * @name image.breakpoints
 		 * @type {number[]}
 		 * @default `[640, 750, 828, 1080, 1280, 1668, 2048, 2560] | [640, 750, 828, 960, 1080, 1280, 1668, 1920, 2048, 2560, 3200, 3840, 4480, 5120, 6016]`
 		 * @description
-		 * The breakpoints used to generate responsive images. Requires the `experimental.responsiveImages` flag to be enabled. The full list is not normally used,
+		 * The breakpoints used to generate responsive images. Requires a value for `layout` to be set. The full list is not normally used,
 		 * but is filtered according to the source and output size. The defaults used depend on whether a local or remote image service is used. For remote services
 		 * the more comprehensive list is used, because only the required sizes are generated. For local services, the list is shorter to reduce the number of images generated.
 		 */
-		experimentalBreakpoints?: number[];
+		breakpoints?: number[];
 		/**
 		 * @docs
-		 * @name image.experimentalDefaultStyles
+		 * @name image.responsiveStyles
 		 * @type {boolean}
 		 * @default `true`
```

Comment:
In the changeset and PR description, you say: `responsiveStyles: true, // This is now *false* by default`, and looking at the defaults in `schema/base.ts` it seems correct. So, I guess there is a mismatch here:
```suggestion
		 * @default `false`
```